samples/Calling/src/app/utils/AppUtils.ts

```diff
@@ -42,21 +42,16 @@ export const getGroupIdFromUrl = (): GroupLocator | undefined => {
 export const createGroupId = (): GroupLocator => ({ groupId: generateGUID() });
 
 /* @conditional-compile-remove(rooms) */
-export const ROOMS_API_BASE_URL = 'http://localhost:7071';
-
-/* @conditional-compile-remove(rooms) */
-export const createRoomId = async (): Promise<string> => {
+/**
+ * Create an ACS room
+ */
+export const createRoom = async (): Promise<string> => {
   const requestOptions = {
-    method: 'POST',
-    headers: {
-      Host: 'alkwa-fn-test.azurewebsites.net',
-      'Content-Type': 'application/json',
-      'Referrer-Policy': 'no-referrer'
-    }
+    method: 'POST'
   };
-  const response = await fetch(ROOMS_API_BASE_URL + '/api/Rooms-CreateRoom', requestOptions);
+  const response = await fetch(`/createRoom`, requestOptions);
   if (!response.ok) {
-    throw 'Invalid token response';
+    throw 'Unable to create room';
```

Comment:
note: always use `throw new Error(msg)` instead of just `throw msg`: https://stackoverflow.com/a/46295341/3047190